README.md

```diff
@@ -26,14 +26,24 @@ class CfgFunctions {
   #include "node_modules\@gruppe-adler\replay\cfgFunctions.hpp"
  };
 ```
+
+```
+class GRAD_Replay {
+  precision = 5;                          // number is precision of replay, 5 means every 5 seconds one snapshot
+  trackedSides = [west, east, civilian];  // defines the sides that will be tracked (possible are west, east, independant, civilian)
```

Comment:
widerspricht der Kommentar grad dem Code? ^^

Reply:
Was meinst du?

Reply:
hmm okay nicht direkt Widerspruch. 
Aber dein Kommentar benennt "independent", die Config schließt das jedoch aus… besser pack alle Möglichkeiten in die Config rein, und laß die entpsrechende Auflistung im Kommentar wegd

Reply:
Das ist ne Readme, die Werte da sind die default Werte. Das würde dann bedeuten ich müsste die Überall angeben,

Reply:
:thinking: aber warum willst du per default nicht auch independent beobachten?